tests/probes_test.go

```diff
@@ -75,7 +68,7 @@ var _ = Describe("[ref_id:1182]Probes", func() {
 			Expect(vmiReady(vmi)).To(Equal(v1.ConditionFalse))
 
 			By("Starting the server inside the VMI")
-			serverStarter(vmi, 1500, useFedoraVM)
+			serverStarter(vmi, 1500, true)
```

Comment:
I might be missing something here; how can you do this ? If there's only IPv4, you don't want to use a big needy fedora VM.

Reply:
Since there is an issue with using cirros VM in this test on dual stack cluster then I think we should be consistent and use the image that work for both IP families instead of checking if a cluster is dual stack.

Eventually all providers will support dual stack and the issue with `netcat` on cirros will remain until we will fix it.
I think we should use only fedora for this test to keep it simple and not switching VM images because one wont work, 
we dont save much resources in this case it wont be noticeable.

Reply:
I haven't explained myself properly. There is no issue with netcat on fedora. You can just plainly use a cirros VM and it will work.

Reply:
There is an issue using netcat with cirros image for this test, isnt it?

Reply:
It's explained better in https://github.com/kubevirt/kubevirt/pull/4129/commits/e956c1a282e2cf036fffd28a91d3977a147dd272

Reply:
So we can use cirros for this test as well, I will change it